docs/product-user-guide/src/main/asciidoc/chap-red-hat-jboss-dashboard-builder.adoc

```diff
@@ -1,83 +1,173 @@
+. Press *Enter* or, or click the image:tick.png[tick symbol].
+. Click *Save* in the pop-up window to save changes to the navigation tree, or *Cancel* to undo the changes.
+
+To check the new submenu is displayed in the Business Central menu, click *Menu*.
+
+
+[id='_dashbuilder-adding-perspective-components_task']
+=== Adding Components to a Custom Perspective 
+
+The Dashboard Builder allows you to create and display custom data in new perspectives that can be added to the Business Central menu.
+
+You can use as many components as required to display custom content. Arrange multiple components in the *Perspective Editor* by clicking and dragging the component into position. 
+
+Use the following procedure add custom Uberfire Widget components to a perspective in the *Perspective Editor*:
+
+.Prerequisite
+* A blank perspective created in the Business Central Dashboard Builder. See xref:_dashbuilder-adding-perspective-to-navigation-tree_task[].
+
+.Procedure
+
+. Open the Dashboard Builder by clicking *Menu* -> *Design* -> *Dashboards* in Business Central.
+. In the *Perspectives* pane on the left, select the custom perspective to be edited.
+. Add a component to the *Perspective Editor*, pane by selecting the component from the list of Uberfire Widgets on the right and dragging it into the editor.
++
+See xref:_dashbuilder-components_con[] for the full list of components and their functions.
+. Click *Save* in the upper-right corner to save the changes to the perspective.
+
+To edit, or to remove a component from the perspective:
+
+. Click the image:gsgEditBtn.png[edit button] in the upper-right corner of the component in the *Perspective Editor*.
+. Select one of the following options:
+* *Edit* to re-open the component widget.
+* *Remove* to remove the component from the perspective.
 
 
-The *Process & Task Dashboard* contains several performance indicators monitoring the jBPM Execution Engine. The data used by the dashboard comes from two tables of the database belonging to the engine: `processinstancelog` and ``bamtasksummary``.
+[id='_dashbuilder-components_con']
+=== Dashboard Builder Components
+
+The Dashboard Builder includes a number of Uberfire Widget components that allow you to create perspectives using custom data. A component can be added to a perspective by dragging it from the list on the right into the *Perspective Editor* pane.
+
+See xref:_dashbuilder-adding-perspective-components_task[] for more information about how to add components to a perspective.
+
+HTML Component::
++
+image:html-component.png[]
++
+The *HTML Component* opens a HTML editor widget. This can be used to easily create HTML pages using text, images, tables, links, colors, and so on. 
+
+Perspective Component::
+The *Perspective Component* adds other custom perspectives to the perspective being edited. This can be used to create a dashboard similar to the xref:_process_and_task_dashboard[].
```

Comment:
"This can be used to create a dashboard similar to the xref:_process_and_task_dashboard[]."
that's a bit misleading. It's not really clear how I could use perspective component to create something like process and task dashboard. To create perspective like process and task dashboard you'd have to use: 1) Tab list component 2) (many) displayer components (tables & charts) and potentially 3) perspective component.

The purpose of "perspective component" is to be able to add (already previously created) existing perspectives into newly created perspectives.

I'd replace the 2nd sentence with something like "You can use this feature to nest Perspectives you already created into newly created perspectives"

Reply:
Changed to:

The *Perspective Component* allows you to add previously created perspectives to a new perspective. You can use this component to nest custom perspectives you have already created in your newly-created perspective.